### PR TITLE
Refactored with bio information

### PR DESCRIPTION
**Pull request by akwong189**
Written by Sophia, just creating a pull request to remove the branch

### CODE REVIEW

File: sample_backend.py [hunk 1]
```diff
@@ -0,0 +1,224 @@
+from flask import Flask,Response
```
Comment (akwong189) [inline sample_backend.py]: Unsure why sample backend is back in the program

File: utils/database.py [hunk 2]
```diff
@@ -187,7 +187,22 @@ def delete_like(self, user_id, article_id):
         )
         return 200
 
-    def update_bio(self, user_id, bio):
+    def update_bio(self, user_id, bio="",first_name="",last_name="", profile_pic="", new_password="", new_email=""):
```
Comment (akwong189) [inline utils/database.py]: this function looks good and easy to understand

File: endpoints/user.py [hunk 3]
```diff
@@ -17,3 +17,26 @@ def update_user_interests():
             current_user["user_id"], data["add"], data["remove"]
         )
         return {"msg": "success"}, 200
+
+@user_blueprint.route("/edit_profile" , methods=["POST"])
```
Comment (akwong189) [inline endpoints/user.py]: still a template for later implementation?

**Review by akwong189 — COMMENTED** [review]
Looks good to be merged

File: endpoints/user.py [hunk 4]
```diff
@@ -17,3 +17,22 @@ def update_user_interests():
...
+
+@user_blueprint.route("/edit_profile", methods=["POST"])
+@jwt_required()
+def update_user_bio():
+    if request.method == "POST":
+        data = request.get_json(force=True)
+        current_user = get_jwt_identity()
+
+    # get the info from each field store it so its attached to the user
+    # shows up everytime
```
Comment (sfparrett) [inline endpoints/user.py:30]: code smell unnecessary comments (this is probably from me I will delete those)

**Review by sfparrett — COMMENTED** [review]
Looks good!

**Pull request merged**